english/cry/cry.md

```diff
@@ -0,0 +1,3 @@
+As you cry and cry, your tears fill the cup you're holding and soon you have a cup full of salty tears. And you can't do much with that!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.